cmd/publish.go

```diff
@@ -56,8 +56,8 @@ dapr publish --enable-domain-socket --publish-app-id myapp --pubsub target --top
 
 		client := standalone.NewClient()
 		// TODO(@daixiang0): add Windows support
-		if unixDomainSocket != "" {
-			if runtime.GOOS == "windows" && publishSocket != "" {
+		if publishSocket != "" {
+			if runtime.GOOS == "windows" {
```

Comment:
port and grpcport must be declared in the var block in the start as int, doesn't seem like they're declared anywhere. Although I would have expected it to show up in the build

Reply:
Do we need them in invoke and publish?

Reply:
not sure if you really need them honestly, maybe just remove them?